src/modules/trainee/types.graphql

```diff
@@ -0,0 +1,10 @@
+type Trainee {
+  id: ID!
+  name: String!
+  email: String!
+}
+
+input UpdateTrainee {
```

Comment:
why we are using different type?
we can use type Trainee and make name and email as non-mandatory fields